fastlane/lib/fastlane/actions/carthage.rb

```diff
@@ -23,10 +23,10 @@ def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :command,
                                        env_name: "FL_CARTHAGE_COMMAND",
-                                       description: "Carthage command (one of `build`, `bootstrap`, `update`)",
+                                       description: "Carthage command (one of `build`, `bootstrap`, `update`, `archive`)",
                                        default_value: 'bootstrap',
                                        verify_block: proc do |value|
-                                         raise "Please pass a valid command. Use one of the following: build, bootstrap, update" unless %w(build bootstrap update).include? value
```

Comment:
Can you [extract](https://github.com/fastlane/fastlane/blob/master/supply/lib/supply/options.rb#L27) the list of values here to tidy up this close a bit? 

Can you also change the raise to `UI.user_error!`? This is the new way of printing errors.